service/src/main/java/uk/gov/hmcts/reform/fpl/service/NoticeOfProceedingsService.java

```diff
@@ -57,46 +56,39 @@
         return removedDocumentBundles.build();
     }
```

Comment:
We can extend this class with DocmosisTemplateDataGeneration to inherit common methods